Fix all warnings and mistakes in documentation

For check warnings and errors user the [DocGeneratorBuilder](https://quasarapp.ddns.net:8043/#/builders/9)
Warnings prints in the " Generate docs for the project " step.

When all warning will be fixed then check docs site using the "copy buildet files" step of the buildbot.

there are still some warnings in the main branch 
```
warning: source ./QtBigInt/res is not a readable file or directory... skipping.
warning: failed to open layout file '/res/DoxyStyle/DoxygenLayout.xml' for reading!
/root/worker/docsGen/DocsGenerator/QuasarApp/Qt-Secret/src/Qt-RSA/qrsaencryption.h:75: warning: The following parameters of QRSAEncryption::generatePairKey(QByteArray &pubKey, QByteArray &privKey, QRSAEncryption::Rsa rsa) are not documented:
/root/worker/docsGen/DocsGenerator/QuasarApp/Qt-Secret/src/Qt-RSA/qrsaencryption.h:90: warning: The following parameters of QRSAEncryption::generatePairKey(QByteArray &pubKey, QByteArray &privKey, const QByteArray &genesis, QRSAEncryption::Rsa rsa) are not documented:
/root/worker/docsGen/DocsGenerator/QuasarApp/Qt-Secret/README.md:156: warning: Illegal command ref as part of a <h4> tag
```

--- a/doxygen.conf
+++ b/doxygen.conf
@@ -1195,7 +1195,7 @@ HTML_STYLESHEET        =
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  = src/mini-gmp/CMake/doxygenStyles.css 
+HTML_EXTRA_STYLESHEET  = src/mini-gmp/CMake/DoxyStyle/doxygenStyles.css 
 
 # The HTML_EXTRA_FILES tag can be used to specify one or more extra images or
 # other source files which should be copied to the HTML output directory. Note

--- a/doxygen.conf
+++ b/doxygen.conf
@@ -703,7 +703,7 @@ FILE_VERSION_FILTER    =
 # DoxygenLayout.xml, doxygen will parse it automatically even if the LAYOUT_FILE
 # tag is left empty.
 
-LAYOUT_FILE            = /res/DoxyStyle/DoxygenLayout.xml
+LAYOUT_FILE            = src/mini-gmp/CMake/DoxygenLayout.xml
 
 # The CITE_BIB_FILES tag can be used to specify one or more bib files containing
 # the reference definitions. This must be a list of .bib files. The .bib
@@ -1133,7 +1133,7 @@ GENERATE_HTML          = YES
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            = html
+HTML_OUTPUT            = https://quasarapp.ddns.net:3031/docs/QuasarApp/Qt-Secret/latest/
 
 # The HTML_FILE_EXTENSION tag can be used to specify the file extension for each
 # generated HTML page (for example: .htm, .php, .asp).

--- a/doxygen.conf
+++ b/doxygen.conf
@@ -703,7 +703,7 @@ FILE_VERSION_FILTER    =
 # DoxygenLayout.xml, doxygen will parse it automatically even if the LAYOUT_FILE
 # tag is left empty.
 
-LAYOUT_FILE            = src/mini-gmp/CMake/DoxygenLayout.xml
+LAYOUT_FILE            = src/mini-gmp/CMake/DoxyStyle/DoxygenLayout.xml
 
 # The CITE_BIB_FILES tag can be used to specify one or more bib files containing
 # the reference definitions. This must be a list of .bib files. The .bib
@@ -1195,7 +1195,7 @@ HTML_STYLESHEET        =
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  = res/DoxyStyle/doxygenStyles.css 
+HTML_EXTRA_STYLESHEET  = src/mini-gmp/CMake/doxygenStyles.css 
 
 # The HTML_EXTRA_FILES tag can be used to specify one or more extra images or
 # other source files which should be copied to the HTML output directory. Note

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -98,7 +98,7 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
      * @param rawData Encode rawData.
      * @param pubKey Public key. Public key size must be equals size of RSA class.
      * @param rsa Encryption key length.
-     * @param blockSizeMode Block size. See BlockSize.
+     * @param blockSizeMode Block size. See the BlockSize enum class.
      * @return The encoded data.
      * @note This is wrapper for a static method.
      */

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -110,7 +110,7 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
      * @param rawData Ecoded data.
      * @param privKey Private RSA key.
      * @param rsa Encryption key length.
-     * @param blockSizeMode Block size mode see BlockSize.
+     * @param blockSizeMode Block size mode see the BlockSize enum class.
      * @return Decoded data.
      * @note This is wrapper for a static method.
      */

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -69,6 +69,7 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
      * @brief generatePairKey Generate RSA pair keys.
      * @param pubKey Return value of pablic key.
      * @param privKey Return value of private key.
+     * @param rsa Encryption key length. See more information in ::QRSAEncryption::
      * @warning  The keys will not be the same if you generate RSA keys of different sizes.
      * @return true if all good.
      */
@@ -80,6 +81,7 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
      * @param pubKey Return value of public key.
      * @param privKey Return value of private key.
      * @param genesis random bytes of arbitrary size.
+     * @param rsa Encryption key length.
      *  This option allows you to generate a key pair that is attached to a given set of bytes.
      *  Example if you set the value to 0xFF, then you will always get the same key pair attached to this value.
      *  This can be convenient if you have some kind of super key by which you want to generate a pair of RSA keys.
@@ -93,42 +95,46 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
 
     /**
      * @brief encode Static implementation of encode.
-     * @param rawData
-     * @param pubKey
-     * @param rsa
-     * @param blockSizeMode
-     * @return
+     * @param rawData Encode rawData.
+     * @param pubKey Public key. Public key size must be equals size of RSA class..
+     * @param rsa Encryption key length.
+     * @param blockSizeMode Block size. See BlockSize.
+     * @return The encoded data.
+     * @note This is wrapper for a static method.
      */
     static QByteArray encode(const QByteArray &rawData, const QByteArray &pubKey,
                               Rsa rsa, BlockSize blockSizeMode = BlockSize::Auto);
 
     /**
      * @brief decode Static implementation of decode.
-     * @param rawData
-     * @param privKey
-     * @param rsa
-     * @param blockSizeMode
-     * @return
+     * @param rawData Ecoded data.
+     * @param privKey Private RSA key.
+     * @param rsa Encryption key length.
+     * @param blockSizeMode Block size mode see BlockSize.
+     * @return Decoded data.
+     * @note This is wrapper for a static method.
      */
     static QByteArray decode(const QByteArray &rawData, const QByteArray &privKey,
                               Rsa rsa, BlockSize blockSizeMode = BlockSize::Auto);
 
     /**
      * @brief signMessage Static implementation of signMessage.
-     * @param rawData
-     * @param privKey
-     * @param rsa
-     * @return
+     * @param rawData The raw data for signature.
+     * @param privKey The private key.
+     * @param rsa Encryption key length.
+     * @return The signed message.
+     * @note This is wrapper for a static method.
      */
     static QByteArray signMessage(QByteArray rawData, const QByteArray &privKey,
                                    Rsa rsa);
 
     /**
      * @brief checkSignMessage Static implementation of checkSignMessage.
-     * @param rawData
-     * @param pubKey
-     * @param rsa
-     * @return
+     * @param rawData The raw data for verification.
+     * @param pubKey The public key.
+     * @param rsa Encryption key length.
+     * @return True if data sigend.
+     * @note This is wrapper for a static method.
      */
     static bool checkSignMessage(const QByteArray &rawData, const QByteArray &pubKey,
                                   Rsa rsa);

--- a/src/Qt-RSA/qrsaencryption.h
+++ b/src/Qt-RSA/qrsaencryption.h
@@ -96,7 +96,7 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
     /**
      * @brief encode Static implementation of encode.
      * @param rawData Encode rawData.
-     * @param pubKey Public key. Public key size must be equals size of RSA class..
+     * @param pubKey Public key. Public key size must be equals size of RSA class.
      * @param rsa Encryption key length.
      * @param blockSizeMode Block size. See BlockSize.
      * @return The encoded data.

